.pre-commit-config.yaml

```diff
@@ -1,21 +1,25 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Can you revert the whitespace-only changes in this file?

Reply:
Will do. Sorry, vs code's auto formatting 🙇‍♂️